docs/profiles.md

```diff
@@ -83,6 +83,8 @@ access.lua        | Function for finding relevant access tags
 destination.lua   | Function for finding relevant destination tags
 maxspeed.lua      | Function for determining maximum speed
 guidance.lua      | Function for processing guidance attributes
+local_trunks.lua  |Function to check trunk roads for access based on country 
```

Comment:
```suggestion
local_trunks.lua  | Function to check trunk roads for access based on country 
```

Reply:
Reopen, not resolved.

Reply:
Hi frodrigo.
I assume the issue is PR #6708 (which resolves PR #6697). I commented on #6697 on 4 Oct while I was working on my issue. I included my changes in my resolution. The issue #6703 (or #6710) addresses the issue where bicycle (and foot) access is allowed on highway="trunk" in some countries.
If #6708 is added the set of countries is reduced. (docs/profiles.md line 400 in this PR).
If #6708 is to be added to Project-OSRM:master I will move this back to draft and then reopen when it is added. 
Ciao
Tom

Reply:
@systemed What do you think of this change request and two PR ?